elements/pl-integer-input/pl-integer-input.py

```diff
@@ -37,9 +37,22 @@ def prepare(element_html, data):
         if name in data['correct_answers']:
             raise Exception('duplicate correct_answers variable name: %s' % name)
         # Test conversion, but leave as string so proper value is shown on answer panel
-        if pl.string_to_integer(correct_answer, base) is None:
+        a_tru_parsed = pl.string_to_integer(correct_answer, base)
+        if a_tru_parsed is None:
             raise Exception('correct answer is not a valid input: %s' % name)
+        if a_tru_parsed > 2 ** 53 - 1 or a_tru_parsed < -((2 ** 53) - 1):
+            raise Exception('answer must be between (2**53)-1 and -((2**53)-1)')
```

Comment:
when saying "between A and B" I think it's clearer if we order them with A < B? And indicate which answer.
```suggestion
            raise Exception('correct answer must be between -(2**53)-1 and ((2**53)-1)')
```

Reply:
For clarity I think we should use `^` for exponentiation and also write the actual values:
```suggestion
            raise Exception('correct answer must be between -9007199254740991 and +9007199254740991 (that is, between -(2^53 - 1) and +(2^53 - 1)).')
```

Reply:
Marking unresolved because the first comment was handled, but not the second.